src/framework/core/js/DataBinding.js

```diff
@@ -417,8 +439,8 @@ var fluid_2_0 = fluid_2_0 || {};
         if (!transId) {
             fluid.fail("Cannot get transaction record without transaction id");
         }
-        var transRec = instantiator.modelTransactions[transId];
-        if (!transRec && !instantiator.free) {
+        var transRec = instantiator && instantiator.modelTransactions[transId];
+        if (!transRec && instantiator) {
```

Comment:
If the statement were written in that form, we'd then have to duplicate the reference expression to assign a value to transRec which is used in the if statement body and return value